docs/API.md

```diff
@@ -796,11 +797,20 @@ __Parameters__
 | ``prefix`` | _String_ | Only listen for objects with the given prefix. |
 | ``suffix`` | _String_ | Only listen for objects with the given suffix. |
 | ``events`` | _String[]_ | Only listen for the specified events, such as s3:ObjectCreated:*, s3:ObjectAccessed:*, s3:ObjectRemoved:*, ..  |
+| ``listener`` | _ BucketEventListener_ | Interface with updateEvent method |
```

Comment:
You have an extra space in between `_`  and `BucketEventListener`
in `_ BucketEventListener_`

Reply:
Corerected